src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -75,9 +77,12 @@ export default {
 			}).then(result => {
 				const savedSearchData = result?.data?.my?.savedSearches;
 				this.totalCount = savedSearchData?.totalCount ?? 0;
-				this.savedSearches = [...savedSearchData.values];
+				this.savedSearches = [...savedSearchData?.values];
 			});
-		}
+		},
+		deleteSavedSearch(payload) {
+			console.log(payload);
```

Comment:
Is this for a future ticket? I don't see this method used anywhere.

Reply:
This method block can be removed for now as it's not in use.